From 30b91a0898e50874886343b66d27f78eaf960faf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 2 Apr 2019 16:54:42 +0200
Subject: [PATCH] EU::MM is not deprecated
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Contrary to the warning in the Makefile.PL, only ExtUtils::MakeMaker
is a full-fledged system. Module::Build is not maintained anymore and
Module::Build::Tiny contains design flaws regarding XS compilation.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 61 -----------------------------------------------------
 1 file changed, 61 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 5915c46..e98af42 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -9,67 +9,6 @@ use warnings;
 use 5.006;
 use ExtUtils::MakeMaker;
 
-BEGIN {
-my %configure_requires = (
-    'ExtUtils::MakeMaker' => '0',
-    'Module::Build::Tiny' => '0.034',
-);
-
-my %errors = map {
-    eval "require $_; $_->VERSION($configure_requires{$_}); 1";
-    $_ => $@,
-} keys %configure_requires;
-
-if (grep { $_ } values %errors)
-{
-    warn "Errors from configure prereqs:\n"
-        . do {
-            require Data::Dumper; Data::Dumper->new([ \%errors ])->Indent(2)->Terse(1)->Sortkeys(1)->Dump;
-        };
-}
-
-if (not $ENV{PERL_MM_FALLBACK_SILENCE_WARNING})
-{
-    warn <<'EOW';
-*** WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING ***
-
-If you're seeing this warning, your toolchain is really, really old* and
-you'll almost certainly have problems installing CPAN modules from this
-century. But never fear, dear user, for we have the technology to fix this!
-
-If you're using CPAN.pm to install things, then you can upgrade it using:
-
-    cpan CPAN
-
-If you're using CPANPLUS to install things, then you can upgrade it using:
-
-    cpanp CPANPLUS
-
-If you're using cpanminus, you shouldn't be seeing this message in the first
-place, so please file an issue on github.
-
-If you're using a packaging tool through a unix distribution, this issue
-should be reported to the package manager.
-
-If you're installing manually, please retrain your fingers to run Build.PL
-when present instead of Makefile.PL.
-
-This public service announcement was brought to you by the Perl Toolchain
-Gang, the irc.perl.org #toolchain IRC channel, and the number 42.
-
-----
-
-* Alternatively, you are doing something overly clever, in which case you
-should consider setting the 'prefer_installer' config option in CPAN.pm, or
-'prefer_makefile' in CPANPLUS, to 'mb" and '0' respectively.
-
-You can also silence this warning for future installations by setting the
-PERL_MM_FALLBACK_SILENCE_WARNING environment variable.
-EOW
-    sleep 10 if -t STDIN && (-t STDOUT || !(-f STDOUT || -c STDOUT));
-}
-} # end BEGIN
-
 my %WriteMakefileArgs = (
   "ABSTRACT" => "A simple http server class",
   "AUTHOR" => "Gisle Aas <gisle\@activestate.com>",
-- 
2.20.1

